server/controllers/user.js

```diff
@@ -14,7 +15,6 @@ class UserController {
       const user = await this.userService.getUser(req.params.id);
       if (user == null) {
         next(ApiError.notFound(`User with id ${userId} not found`));
-        return;
       }
       res.status(200).json(user);
```

Comment:
Will the removed return statement result in this line now being reached (and incorrectly giving a 200 response), or would that be prevented by the next() call?

Reply:
It will be prevented by the `next()` call. The error handler middleware consists of `res.status(...).json(...)` inside, so `res.status(200).json(user)` here would not be reached.